turbiniactl

```diff
@@ -24,6 +24,7 @@ from datetime import timedelta
 import json
 import logging
 import sys
+import time
 
 try:
   import psq
```

Comment:
qq: do we need this import check here? Isn't psq a required dependency?

Reply:
Yeah, it is a required dependency.  I think it's just here to give a nicely formatted message back to the user.  Maybe we could change it to point to the install instructions rather than the PSQ page though.  I'll do that next time I touch this file.